rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -96,13 +99,23 @@
 @ApplicationScoped
 @Path(COORDINATOR_PATH_NAME)
 @Tag(name = "LRA Coordinator")
-public class Coordinator {
-
+public class Coordinator extends Application {
```

Comment:
It can be either, historically the resource had always extended Application. I moved the initialisation into the resource because Undertow didn't work with a separate Application class.

Reply:
Would not be that the Undertow requires only the `getClasses` method to be defined. If there is one initialization class where the `getClasses` points to both resource then it could work (maybe).

Reply:
There are two context roots.